src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -251,19 +251,22 @@ export class CalcitePickList {
 
   render() {
     const { disabled, loading } = this;
+    const hasMenu = this.el.querySelector("[slot=menu-actions]");
     return (
       <Host aria-disabled={disabled} aria-busy={loading}>
-        <header>
-          {this.filterEnabled ? (
-            <calcite-filter
-              data={this.dataForFilter}
-              textPlaceholder={TEXT.filterPlaceholder}
-              aria-label={TEXT.filterPlaceholder}
-              onCalciteFilterChange={this.handleFilter}
-            />
-          ) : null}
-          <slot name="menu-actions" />
-        </header>
+        {hasMenu ? (
```

Comment:
This will not render the filter in the event filter-enabled is true but there's no menu-actions slot. Not sure the intention but I'm pretty sure we don't want that side-effect.

Reply:
Looks like in the other file you're using `filterEnabled || hasMenu ?` which is probably what you want here.

Reply:
Ah! Good catch.

Reply:
Thanks @pr3tori4n. Fixed.